ui/site/css/ublog/_post.scss

```
@@ -64,6 +64,7 @@
     font-weight: bold;
     cursor: pointer;
     &::before {
+      margin-#{$start-direction}: 0.4em;
       margin-#{$end-direction}: 0.4em;
```

```suggestion
      margin: 0 0.4em;
```

If the `margin-left` and `margin-right` are the same for LTR and RTL languages, can we consolidate?